sdk/python/tests/compiler/container_builder_test.py

```diff
@@ -58,10 +58,32 @@ def test_generate_kaniko_yaml(self, mock_gcshelper):
     test_data_dir = os.path.join(os.path.dirname(__file__), 'testdata')
 
     # check
-    builder = ContainerBuilder(gcs_staging=GCS_BASE, default_image_name=DEFAULT_IMAGE_NAME, namespace='default')
+    builder = ContainerBuilder(gcs_staging=GCS_BASE,
+                               default_image_name=DEFAULT_IMAGE_NAME,
+                               namespace='default')
     generated_yaml = builder._generate_kaniko_spec(docker_filename='dockerfile',
-                                                   context='gs://mlpipeline/kaniko_build.tar.gz', target_image='gcr.io/mlpipeline/kaniko_image:latest')
+                                                   context='gs://mlpipeline/kaniko_build.tar.gz',
+                                                   target_image='gcr.io/mlpipeline/kaniko_image:latest')
     with open(os.path.join(test_data_dir, 'kaniko.basic.yaml'), 'r') as f:
       golden = yaml.safe_load(f)
 
+    self.assertEqual(golden, generated_yaml)
+
+  def test_generate_kaniko_yaml_kubeflow(self, mock_gcshelper):
+    """ Test generating the kaniko job yaml for Kubeflow deployment """
+
+    # prepare
+    test_data_dir = os.path.join(os.path.dirname(__file__), 'testdata')
+
+    # check
+    builder = ContainerBuilder(gcs_staging=GCS_BASE,
+                               default_image_name=DEFAULT_IMAGE_NAME,
+                               namespace='user'
```

Comment:
Missing comma at the end of the line.

P.S. It's possible to put comma even after the last argument so that arguments can be added, removed or moved without thinking about commas.

Reply:
Got it, I wasn't familiar with python style guide. If possible, can we have a guide for setting up auto formatting? I currently don't know how to set it up with vscode.